codespell_lib/data/dictionary.txt

```diff
@@ -19338,6 +19527,7 @@ usueful->useful
 ususal->usual
 ususally->usually
 UTF8ness->UTF-8-ness
+uti->uri
```

Comment:
i worry about 3 letter typos like this that will trigger false positives. Maybe designate it as a warning instead?

Reply:
removed